pkg/controller/dynamodb/table/hooks.go

```diff
@@ -308,12 +328,33 @@ func (e *updateClient) preUpdate(_ context.Context, cr *svcapitypes.Table, u *sv
 	case cr.Spec.ForProvider.StreamSpecification != nil &&
 		(awsgo.BoolValue(t.Table.StreamSpecification.StreamEnabled) != awsgo.BoolValue(cr.Spec.ForProvider.StreamSpecification.StreamEnabled)):
 		newUpdateObj.StreamSpecification = u.StreamSpecification
-	default:
-		return errors.New("only provisionedThroughput and streamSpecification updates are supported")
+
+	case cr.Spec.ForProvider.SSESpecification != nil &&
+		(aws.StringValue(t.Table.SSEDescription.KMSMasterKeyArn) != aws.StringValue(cr.Spec.ForProvider.SSESpecification.KMSMasterKeyID)) &&
+		(aws.StringValue(t.Table.SSEDescription.SSEType) == aws.StringValue(cr.Spec.ForProvider.SSESpecification.SSEType)):
+		newUpdateObj.SSESpecification = u.SSESpecification
+
+	case cr.Spec.ForProvider.BillingMode != nil &&
+		cr.Spec.ForProvider.BillingMode != t.Table.BillingModeSummary.BillingMode:
+		newUpdateObj.BillingMode = u.BillingMode
+
+		// TODO(muvaf): ReplicationGroupUpdate and GlobalSecondaryIndexUpdate features
+		// are not implemented yet.
 	}
-	// TODO(muvaf): ReplicationGroupUpdate and GlobalSecondaryIndexUpdate features
-	// are not implemented yet.
 
 	*u = *newUpdateObj
+
 	return nil
 }
+
+func postUpdate(_ context.Context, cr *svcapitypes.Table, t *svcsdk.UpdateTableOutput, upd managed.ExternalUpdate, err error) (managed.ExternalUpdate, error) {
+	if err != nil {
+		// no update required - newUpdateObj was empty
+		// no enum available for this skip
+		if strings.Contains(err.Error(), "ProvisionedThroughput, BillingMode, UpdateStreamEnabled, GlobalSecondaryIndexUpdates or SSESpecification or ReplicaUpdates is required") {
```

Comment:
Would a normal user hit this? is it possible to do something about this in `isUpToDate`?

Reply:
@chlunde at the moment we have preUpdate with switch/case and creating one specific Update object with one of the mentioned keys - if nothing is to-do we receive this Message that we need to send one update  for the mentioned key- If you have a better way via IsUpToDate please make a suggestion

Reply:
@chlunde Any ideas for isUpToDate ? Or can we Go with the PR solution ?